data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/ledger-utxo-creation-v5.1.xml

```diff
@@ -3,18 +3,106 @@
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
 
+    <property name="json.column.type" value="JSONB" dbms="postgresql"/>
+
+    <!-- Please note that HSQLDB schema is supported for integration test purposes only. -->
+    <property name="json.column.type" value="CLOB" dbms="hsqldb"/>
```

Comment:
Are these not already available via `ledger-utxo-visible-transaction-state-creation-v1.0.xml`?

Reply:
I haven't tried it yet tbh, will give it a go

Reply:
just did a start-up and it seems like the schema is created without these properties in this XML so will remove. I suppose they are already present from the other changeset as you said.